lib/govuk_ab_testing/rspec_helpers.rb

```diff
@@ -0,0 +1,39 @@
+module GovukAbTesting
```

Comment:
If we're doing this properly, I think I would personally prefer one more layer of abstraction that would keep the logic synced up between these two test helpers. For example, you might have

```ruby
framework_helper = framework_helper_for(:rspec)

def with_variant(args)
  # Set up variant and AB Test... (omitted)
  framework_helper.set_header(...) # This is implemented by both version of the framework helpers
  
  yield

  framework_helper.assert_vary_header_set(...)
  # etc.
end
```